peer/peer.go

```diff
@@ -197,6 +200,9 @@ type MessageListeners struct {
 	// message.
 	OnSendHeaders func(p *Peer, msg *wire.MsgSendHeaders)
 
+	// OnSendAddrV2 is invoked when a peer receives a sendaddrv2 message.
+	OnSendAddrV2 func(p *Peer, msg *wire.MsgSendAddrV2)
```

Comment:
I don't see us ever specify this in the server peer?

Reply:
This is only for tests, there's not really a reason to use it otherwise but thought I'd add it for completeness.